content/docs/latest/spire-helm-charts-hardened-about/recommendations.md

```diff
@@ -0,0 +1,94 @@
+them applied to your deployment. If there are particular recommmendations
+you do not wish to use, you can still set those recommendations to `false`
+to disable them.
+
+your-values.yaml snippet:
+```yaml
+global:
+  spire:
+    recommendations:
+      enabled: true
+```
+
+# Individual Recommentations
+
+| Name                                                                                                                                | Value                                          |
+| ----------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------- |
+| [Namespace Layout](/docs/latest/spire-helm-charts-hardened-about/recommendations/#namespace-layout)                                 | global.spire.recommendations.namespaceLayout   |
+| [Namespace Pod Security Standards](/docs/latest/spire-helm-charts-hardened-about/recommendations/#namespace-pod-security-standards) | global.spire.recommendations.namespacePSS      |
+| [PriorityClassName](/docs/latest/spire-helm-charts-hardened-about/recommendations/#priority-class-name)                             | global.spire.recommendations.priorityClassName |
+| [Prometheus](/docs/latest/spire-helm-charts-hardened-about/recommendations/#prometheus)                                             | global.spire.recommendations.prometheus        |
+| [Strict Mode](/docs/latest/spire-helm-charts-hardened-about/recommendations/#namespace-layout)                                      | global.spire.recommendations.strictMode        |
+| [Security Contexts](/docs/latest/spire-helm-charts-hardened-about/recommendations/#security-contexts)                               | global.spire.recommendations.securityContexts  |
+
+## Namespace Layout
+
+Option `global.spire.recommendations.namespaceLayout` causes the services to be deployed
+across the two recommended namespaces for the services:
+
+| Namespace Type | Namespace Value                     | Default Value | Purpose                                                    |
+| -------------- | ----------------------------------- | ------------- | ---------------------------------------------------------- |
+| Server         | global.spire.namespaces.server.name | spire-server  | Services that should have restricted Kubernetes privileges |
+| System         | global.spire.namespaces.system.name | spire-system  | Services needing Kubernetes privileges                     |
+
+| Service Name                   | Namespace Type |
+| ------------------------------ | -------------- |
+| SPIRE Server                   | Server         |
+| SPIFFE OIDC Discovery Provider | Server         |
+| SPIFFE CSI Driver              | System         |
+| SPIRE Agent                    | System         |
+
+## Namespace Pod Security Standards
+
+Option `global.spire.recommendations.namespacePSS` sets the chart to set the recommended
```

Comment:
Does this mean that this recommendation sets the values on the table below to `true` instead of using the default values?

Reply:
Hmmm... I guess its unclear, but not really sure how to reword... maybe you could help.

Part of the confusion is I just found a typo I think too.

global.spire.namespaces.spire should have been global.spire.namespaces.server in the docs here. fixed.

Let me try and explain with maybe a more concrete couple of examples...

If you have:
 * global.spire.recommendations.enabled=true (non default)
 * global.spire.recommendations.enabled=true (default)
 * any one of global.spire.namespaces.create=true (non default), global.spire.namespaces.server.create=true (non default), global.spire.namespaces.system.create=true (non default)
 
so, for example:
 ```
global:
  spire:
    recommendations:
      enabled: true
    namespaces:
      create: true
```
 
 It will create namespace(s) with the recommended pod security standard labels applied to it.
 
 In the example  there, namespaces.create=true creates both namespaces, so both will get created, and labeled.
 
Another example:
 ```
global:
  spire:
    recommendations:
      enabled: true
    namespaces:
      system:
        name: myown-system
        create: true
```

would create just the system namespace named 'myown-system' and apply the privileged label to it, but not create nor label the spire-server namespace.

Reply:
Oh I got it all wrong 😆 The first table confused me. Now it makes sense.

What do you think about moving the namespaces table to the Namespaces page, and just linking to it from here.

Reply:
Yeah, I think that might help. Done.